doc/getting-started/acrn-sanity-test.rst

```diff
@@ -0,0 +1,349 @@
+
+IO
+**
+
+ST_ACRNIO_LaaG_Virtio_net
+*************************
+
+* Description
+
+This test case focus on checking if Service OS and LaaG can be allocated different IP, 
+and ping is ok by sharing same physical network card.
+
+* Prerequisites
+
+Need support Virtio, and bridge/tap is created normally in Service OS
+
+* Test Steps
+
+1. Boot DUT, and enter SOS from ACRN hypervisor
+
+   .. code-block:: none
+
+      $ vm_console
+
+#. Check IP and bridge/tap after log in sos
+
+   .. code-block:: none
+     
+      $ ip a
+      $ brctl show
+      bridge name     bridge id               STP enabled     interfaces
+      acrn-br0         8000.1ad4ac453b20       no              acrn_tap0
+                                                               ens4
+
+#. Boot UOS using the launch_uos.sh 
+
+   .. code-block:: none
+   
+      $ bash /usr/share/acrn/samples/nuc/launch_uos.sh
+
+#. Check UOS IP address
+
+* Expected Result
+
+IP address should be allocated for both SOS and UOS, and ping is ok
+
+ST_ACRNIO_LaaG_Virtio_storage
+*****************************
+
+* Description
+
+This test case focus on checking if LaaG using virtio storage can boot up successfully.
+
+* Prerequisites
+
+Need support Virtio
+
+* Test Steps
+
+1. In SOS: copy or create one img file as rootfs.
+#. Boot UOS using this img file in cmd line in launch_uos.sh
+
+   .. code-block:: none
+
+   $ cd /home/clear/uos/
+
+   $ wget https://download.clearlinux.org/releases/xxxxx/clear/clear-xxxxx-kvm.img.xz
+
+   $ unxz clear-xxxxx-kvm.img.xz
+
+   $ vim launch_uos.sh
+
+   -s 3,virtio-blk,/home/clear/uos/uos.img \
+   
+   to
+
+   -s 3,virtio-blk,/home/clear/uos/xxxxx.img \
+
+* Expected Result
+
+UOS can boot up successfully
+
+ST_ACRNIO_SOS_Ethernet
+**********************
+
+* Description
+
+This test case focus on checking if IP can be allocated to Service OS correctly.
+
+* Prerequisites
+
+Network card is installed
+
+* Test Steps
+
+1. Boot into SOS
+#. IP a in SOS
+
+* Expected Result
+
+#. IP address (either local or external) should be allocated for sos(1 ip address), and ping is ok.
+
+ST_ACRNIO_SOS_Storage
+*********************
+
+* Description
+
+This test case focus on checking if EMMC can work normally in Service OS.
+
+* Prerequisites
+
+EMMC is installed
+
+* Test Steps
+
+1. Boot into SOS
+#. fdisk -l in SOS
+#. Find EMMC block and mount it to read/write
+   NVME disk in /dev/mmcblk1p3
+   SATA disk in /dev/sda3
+
+* Expected Result
+
+EMMC can be used normally in SOS
+
+GVT
+***
+
+ST_ACRNGT_SOS_Rendering_Weston_APP
+**********************************
+
+* DescriptioACRNCRNGT provide virtual display on LaaG: run weston /weston-simple-egl in LaaG,
+workload will show normally on LaaG
+
+* Prerequisites
+
+System boot up normally
+
+Install ias weston related bundle
+
+   .. code-block:: none
+
+      $ swupd bundle-add software-defined-cockpit x11-server --allow-insecure-http
```

Comment:
You need to run `swupd` as root (with `sudo`).

Also curious why you need to use the `--allow-insecure-http` flag?